src/base/misc_string.h

```diff
@@ -41,6 +41,9 @@ extern "C"
   extern char *ustr_upper (char *s);
   extern char *ustr_lower (char *s);
 
+  extern int ansisql_strcasecmp (const char *s, const char *t);
+  extern int ansisql_strcmp (const char *s, const char *t);
```

Comment:
ansisql_strcmp is only used at extendible-hash routine in "storage/extendible_hash.c".
therefore, the ansisql_strcmp looks better to move to it as a static function.  and it needs to rename the function name  if it's necessary.
ansisql_strcasecmp is only used at db_query_get_tuple_value_by_name in "compat/db_query.c".
therefore, the ansisql_strcasecmp looks better to move to it as a static function. and it needs to rename the function name if it's necessary.

Reply:
Is there any possibility that they are used in the other modules? The ansisql_XXX functions deal with trailing spaces. Maybe in parsing or query modules, this function could be used in the future.